grapple/types/tags.py

```diff
@@ -3,13 +3,13 @@
 from graphene_django.converter import convert_django_field
 from grapple.utils import resolve_queryset
 
-from modelcluster.contrib.taggit import ClusterTaggableManager
```

Comment:
question: what prompted this change?

Reply:
The `ClusterTaggableManager` extends the `TaggableManager` class, so by registering the base `TaggableManager` class we get proper values on the resolvers for all tag fields.

Reply:
🤔 this'll probably fail on previews as modelcluster allow for that without objects being saved in the database

Reply:
Hum, that's a good point. Maybe we can register both managers?

Reply:
hmm,

tried 
```
@convert_django_field.register(TaggableManager)
@convert_django_field.register(ClusterTaggableManager)
def convert_tag_manager_to_string(field, registry=None):
    return TagObjectType(
```

locally and it doesn't explode. so let's go with that

Reply:
Updated!